### ISSUE TITLE
Items stacked on top of each other

### ISSUE THREAD

**Issue by studiolam** [tsuyoshiwada/react-stack-grid#10]
Hello, I'm using the React-stack-grid library for my website.

Everything works fine, but only on mobile screen every item is stack of top of each other. I analyzed the items and it is specifically when the window is less then 444px. If it is less, they are stack on top of each other. When it is more then 445px, they work just fine.

Does somebody know how to fix this issue? I already tried a lot, like the viewport meta tag, checking if somewhere in my code i used 444 of 445 px, ...

Thanks!

**Comment by damechen** [comment]
Same issue here!

**Comment by laraferroni** [comment]
Had the same thing. If you set a minWidth on the parent container object to at least the column width, it no longer stacks.

**Comment by nimran** [comment]
same issue here. any solution available?

**Comment by jigarvishwakarma** [comment]
Facing the same issue

**Comment by somaria** [comment]
@laraferroni , you have a solution?  could you show us how to set the parent container?

**Comment by laraferroni** [comment]
Sure, it would be something like, where classes home has a width set. In my case, I just set width=100%, but you could also use a minWidth that is greater than 350. 

 ```
<Container className={classes.home}>

    <StackGrid
            columnWidth={350}
            appear={scaleDown.appear}
            appeared={scaleDown.appeared}
            enter={scaleDown.enter}
            entered={scaleDown.entered}
            leaved={scaleDown.leaved}
          >
</Container>
```

**Comment by somaria** [comment]
Yep, manage to get it to work as shown here:  https://todayinpicture.com/

Sometimes, if I refresh the page, the images stacked up on each others.   But if I click on InPictures menu item, it will rearrange to its proper location.

I wonder if there is a way that I could resolve this.

**Comment by loveakinlesi** [comment]
Had the same issue on both laptop and mobile. If you set `monitorImagesLoaded={true}`, it fixes it.  It's actually stated in the documentation [here](https://github.com/tsuyoshiwada/react-stack-grid#performance-when-using-images).
